test/functional/client-ca.test.js

```diff
@@ -62,7 +63,9 @@ test('correctly use supplied CA cert on client for connections', t => {
 
         // Specify CA file
         process.env.CA_CERT = '../certs/ca/my-root-ca.crt.pem';
-        client = app.main({ port: port() });
+        process.env.BROKER_CLIENT_VALIDATION_URL = `https://localhost:${echoServerPort}/test`;
```

Comment:
how did the test pass before your fix?

Reply:
That URL is used only the for /systemcheck endpoint, which was never used in tests. With correct CA setup, Broker works, but if you manually invoke /systemcheck, you get an error.